en/docs/learn/analytics/adding-third-party-non-osgi-libraries.md

```diff
@@ -0,0 +1,15 @@
+# Adding Third Party Non OSGi Libraries
+
+The API Manager Analytics distribution is OSGi-based. Therefore, when you integrate third party products such as MySQL with the API Manager Analytics, you need to check whether the libraries you need to add to the Streaming Integrator are OSGi-based. If they are not, you need to convert them to OSGi bundles before adding them to the `<API-M_ANALYTICS_HOME>/lib` directory.
+
+To convert jar files to OSGi bundles, follow the procedure given below:
```

Comment:
jar should be JAR

Reply:
Fixed in d9c29843ef4201cb9eca588053bea4aaecd174e0